yangcatalog-ui/src/app/features/yang-validator/validation-error/validation-error.component.html

```diff
@@ -1 +1,2 @@
-<ngb-alert type="danger" (close)="onCloseError()" *ngIf="error">{{ error.message || 'The application has encountered an unknown error.'}}</ngb-alert>
+<ngb-alert type="info" (close)="onCloseError()" *ngIf="error">{{ error.message || 'The application has encountered an
+    unknown error.'}}</ngb-alert>
```

Comment:
I am not sure whether we want to completely replace danger alert for info alert.
Because if there will be some other error (not that related to no modules extracted) - then it will be placed in info alert.
I suggest to update payload on backend and not send this message as an error, because it is not error.
I would send it probably as info OR warning.